pizza_cutter_masking/apply_masks.py

```diff
@@ -0,0 +1,48 @@
+def apply_masks(ra, dec, footprint=None, mask=None, metadetect=None):
+    """
+    Get the indices that are within the footprint and not masked by the mask.
+    Optionally check the metadetect mask as well
+
+    Parameters
+    ----------
+    ra: number/array
+        The right ascension in degrees
+    dec: number/array
+        The decliiatioj in degrees
+    footprint: HealSparseMap, optional
+        The healsparse map for the footprint
+    mask: HealSparseMap, optional
+        The healsparse map for the mask
+    metadetect: HealSparseMap, optional
+        The optional healsparse map for the metadetect footprint
```

Comment:
It would be useful to add what should be stored in these masks (bits, true vs false, etc.)

Reply:
this method is actually out of data (it is only used for the demo)

Reply:
"out of date"

Reply:
removed